### ISSUE TITLE
Symbols not loading

### ISSUE THREAD

**Issue by nolancasimir** [microsoft/vscode-cpptools#8919]
Bug type: General

The past week or two, I cannot get any symbols to load. The strip that shows the current path->file->symbol currently is missing the symbol. Control click to peek is not functioning, and the outline for files are empty even for huge files with lots of functions.

Steps to reproduce
1. Open a folder
2. Open a .c file that contains functions
3. Open the explorer window, and notice the message "No symbols found in document 'xyz.c'" in the outline area.

Expected results
1. Open a folder
2. Open a .c file that contains functions
3. Open the explorer window, and notice the symbols for the file (functions etc.) are shown in the outline area.

![image](https://user-images.githubusercontent.com/56938681/155410549-de0ca7fb-9916-45ab-b97b-5d9d6761f40b.png)
![image](https://user-images.githubusercontent.com/56938681/155410860-39a3c5e8-e76b-40f6-85cd-dc1ef13b82a9.png)

I am developing in C. This just started happening recently. Is there any file I need to delete, or memory pool I need to increase to get symbols back?

Thanks,
Nolan

VS Code version: Code 1.64.2 (https://github.com/microsoft/vscode/commit/f80445acd5a3dadef24aa209168452a3d97cc326[](https://github.com/vscode-triage-bot)[](https://github.com/jrieken)[](https://github.com/vscode-triage-bot)[](https://github.com/vscode-triage-bot)[](https://github.com/vscode-triage-bot)[](https://github.com/nolancasimir)[](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)[](https://github.com/jrieken)[](https://github.com/jrieken)[](https://github.com/nolancasimir)[](https://github.com/vscode-triage-bot)[](https://github.com/), 2022-02-09T22:02:28.252Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No

System Info
Item	Value
CPUs	Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz (36 x 3000)
GPU [Status](https://www.githubstatus.com/)	2d_canvas: enabled
gpu_compositing: enabled
multiple_raster_threads: enabled_on
oop_rasterization: enabled
opengl: enabled_on
rasterization: enabled
skia_renderer: enabled_on
video_decode: enabled
vulkan: disabled_off
webgl: enabled
webgl2: enabled
Load (avg)	undefined
Memory (System)	63.72GB (40.16GB free)
Process Argv	--crash-reporter-id e8f7371b-43ab-42b8-8cdf-18b03c590cad
Screen Reader	no
VM	0%
Extensions (32)
Extension	Author (truncated)	Version
better-comments	aar	2.1.0
All-Autocomplete	Ati	0.0.23
code-gnu-global	aus	0.2.2
indented-block-highlighting	byi	1.0.7
arm	dan	1.7.3
xml	Dot	2.5.1
gitlens	eam	11.7.0
vscode-npm-script	eg2	0.3.24
sort-json-array	fvc	3.1.0
aviation-aviation-gifd3-autogen-filter	gar	1.0.0
aviation-c-block-comments	gar	1.0.1
aviation-test-utils	gar	0.0.2
garmin-c-snippets	Gar	1.3.1
gboost-test-explorer	Gar	0.1.1
private-extension-manager	gar	1.8.0
private-extension-manager-remote-helper	gar	1.8.0
vscode-ants	Gar	1.4.1
vscode-test-explorer	hbe	2.21.1
better-cpp-syntax	jef	1.15.13
git-graph	mhu	1.30.0
python	ms-	2022.0.1814523869
vscode-pylance	ms-	2022.2.3
jupyter	ms-	2022.1.1301854968
jupyter-keymap	ms-	1.0.0
jupyter-renderers	ms-	1.0.6
cpptools	ms-	1.8.4
cpptools-extension-pack	ms-	1.1.0
test-adapter-converter	ms-	0.1.5
vsliveshare	ms-	1.0.5330
vsliveshare-audio	ms-	0.1.91
trailing-spaces	sha	0.3.1
debug	web	0.25.1
(1 theme extensions excluded)

A/B Experiments
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392cf:30425750
pythontb:30283811
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
vscorecescf:30438341
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstartedc:30433508
vsclayoutctrc:30437038
vsrem710cf:30416617
dsvsc009:30440023
vsbas813:30436447
vscscmwlcmc:30438804
vscgsvid2:30438800
helix:30440343

**Comment by nolancasimir** [comment]
If I revert back to version 1.7.1 it fixes the issue.

**Comment by Colengms** [comment]
Hi @nolancasimir .  Could you try adding the setting `"C_Cpp.loggingLevel": "Debug"`, reloading the window, and providing the output to the `C/C++` output channel leading up to (and including) repro'ing the issue?  That may provide enough information to investigate further.

**Comment by nolancasimir** [comment]
I'll try and paste as much as I can without getting in trouble with my company

loggingLevel: Debug
cpptools/didChangeCppProperties
Attempting to get defaults from C compiler in "compilerPath" property: 'C:\Program Files\LLVM\12.0.0\bin\clang.exe'
Querying compiler for default C++ language standard using command line: "C:\\Program Files\\LLVM\\12.0.0\\bin\\clang.exe" -x c++ -E -dM nul
Detected language standard version: c++14
Querying compiler for default C language standard using command line: "C:\\Program Files\\LLVM\\12.0.0\\bin\\clang.exe" -x c -E -dM nul
Detected language standard version: c17
Querying compiler's default target using command line: "C:\Program Files\LLVM\12.0.0\bin\clang.exe" -dumpmachine
Compiler returned default target value: x86_64-pc-windows-msvc

Compiler query command line: "C:\\Program Files\\LLVM\\12.0.0\\bin\\clang.exe" -std=c17 -Wp,-v -E -dM -x c -m64 -fno-blocks nul
Code browsing service initialized
Attempting to get defaults from C++ compiler in "compilerPath" property: 'C:\Program Files\LLVM\12.0.0\bin\clang.exe'
Compiler query command line: "C:\\Program Files\\LLVM\\12.0.0\\bin\\clang.exe" -std=c++17 -Wp,-v -E -dM -x c++ -m64 -fno-blocks nul
  Folder: C:/PROGRAM FILES (X86)/MICROSOFT VISUAL STUDIO/2019/PROFESSIONAL/VC/TOOLS/MSVC/14.29.30037/ATLMFC/INCLUDE/ will be indexed
  Folder: C:/PROGRAM FILES (X86)/MICROSOFT VISUAL STUDIO/2019/PROFESSIONAL/VC/TOOLS/MSVC/14.29.30037/INCLUDE/ will be indexed
  Folder: C:/PROGRAM FILES (X86)/WINDOWS KITS/10/INCLUDE/10.0.18362.0/SHARED/ will be indexed
  Folder: C:/PROGRAM FILES (X86)/WINDOWS KITS/10/INCLUDE/10.0.18362.0/UCRT/ will be indexed
  Folder: C:/PROGRAM FILES (X86)/WINDOWS KITS/10/INCLUDE/10.0.18362.0/UM/ will be indexed
  Folder: C:/PROGRAM FILES (X86)/WINDOWS KITS/10/INCLUDE/10.0.18362.0/WINRT/ will be indexed
  Folder: C:/PROGRAM FILES/LLVM/12.0.0/LIB/CLANG/12.0.0/INCLUDE/ will be indexed
Folder: --- will be indexed
textDocument/didOpen: ---
Discovering files...
cpptools/activeDocumentChange: ---
cpptools/getDocumentSymbols: --- (id: 2)
cpptools/getCodeActions: --- (id: 3)
cpptools/getDocumentSymbols
  Processing folder (recursive): C:/PROGRAM FILES (X86)/MICROSOFT VISUAL STUDIO/2019/PROFESSIONAL/VC/TOOLS/MSVC/14.29.30037/ATLMFC/INCLUDE/
  Processing folder (recursive): C:/PROGRAM FILES (X86)/MICROSOFT VISUAL STUDIO/2019/PROFESSIONAL/VC/TOOLS/MSVC/14.29.30037/INCLUDE/
  Processing folder (recursive): C:/PROGRAM FILES (X86)/WINDOWS KITS/10/INCLUDE/10.0.18362.0/SHARED/
  Processing folder (recursive): C:/PROGRAM FILES (X86)/WINDOWS KITS/10/INCLUDE/10.0.18362.0/UCRT/
  Processing folder (recursive): C:/PROGRAM FILES (X86)/WINDOWS KITS/10/INCLUDE/10.0.18362.0/UM/
Checking for syntax errors: ---
  Processing folder (recursive): C:/PROGRAM FILES (X86)/WINDOWS KITS/10/INCLUDE/10.0.18362.0/WINRT/
  Processing folder (recursive): C:/PROGRAM FILES/LLVM/12.0.0/LIB/CLANG/12.0.0/INCLUDE/
  Processing folder (recursive): ---
sending compilation args for ---
  include: C:\PROGRAM FILES\LLVM\12.0.0\LIB\CLANG\12.0.0\INCLUDE
  include: C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2019\PROFESSIONAL\VC\TOOLS\MSVC\14.29.30037\INCLUDE
  include: C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2019\PROFESSIONAL\VC\TOOLS\MSVC\14.29.30037\ATLMFC\INCLUDE
  include: C:\PROGRAM FILES (X86)\WINDOWS KITS\10\INCLUDE\10.0.18362.0\UCRT
  include: C:\PROGRAM FILES (X86)\WINDOWS KITS\10\INCLUDE\10.0.18362.0\SHARED
  include: C:\PROGRAM FILES (X86)\WINDOWS KITS\10\INCLUDE\10.0.18362.0\UM
  include: C:\PROGRAM FILES (X86)\WINDOWS KITS\10\INCLUDE\10.0.18362.0\WINRT
define: _INTEGRAL_MAX_BITS=64
  define: _MSC_BUILD=1
  define: _MSC_EXTENSIONS=1
  define: _MSC_FULL_VER=192930038
  define: _MSC_VER=1929
  define: _M_AMD64=100
  define: _M_X64=100
  define: _WIN32=1
  define: _WIN64=1
  define: __ATOMIC_ACQUIRE=2
  define: __ATOMIC_ACQ_REL=4
  define: __ATOMIC_CONSUME=1
  define: __ATOMIC_RELAXED=0
  define: __ATOMIC_RELEASE=3
  define: __ATOMIC_SEQ_CST=5
  define: __BIGGEST_ALIGNMENT__=16
  define: __BYTE_ORDER__=__ORDER_LITTLE_ENDIAN__
  define: __CHAR16_TYPE__=unsigned short
  define: __CHAR32_TYPE__=unsigned int
  define: __CHAR_BIT__=8
  define: __CLANG_ATOMIC_BOOL_LOCK_FREE=2
  define: __CLANG_ATOMIC_CHAR16_T_LOCK_FREE=2
  define: __CLANG_ATOMIC_CHAR32_T_LOCK_FREE=2
  define: __CLANG_ATOMIC_CHAR_LOCK_FREE=2
  define: __CLANG_ATOMIC_INT_LOCK_FREE=2
  define: __CLANG_ATOMIC_LLONG_LOCK_FREE=2
  define: __CLANG_ATOMIC_LONG_LOCK_FREE=2
  define: __CLANG_ATOMIC_POINTER_LOCK_FREE=2
  define: __CLANG_ATOMIC_SHORT_LOCK_FREE=2
  define: __CLANG_ATOMIC_WCHAR_T_LOCK_FREE=2
  define: __CONSTANT_CFSTRINGS__=1
  define: __DBL_DECIMAL_DIG__=17
  define: __DBL_DENORM_MIN__=4.9406564584124654e-324
  define: __DBL_DIG__=15
  define: __DBL_EPSILON__=2.2204460492503131e-16
  define: __DBL_HAS_DENORM__=1
  define: __DBL_HAS_INFINITY__=1
  define: __DBL_HAS_QUIET_NAN__=1
  define: __DBL_MANT_DIG__=53
  define: __DBL_MAX_10_EXP__=308
  define: __DBL_MAX_EXP__=1024
  define: __DBL_MAX__=1.7976931348623157e+308
  define: __DBL_MIN_10_EXP__=(-307)
  define: __DBL_MIN_EXP__=(-1021)
  define: __DBL_MIN__=2.2250738585072014e-308
  define: __DECIMAL_DIG__=__LDBL_DECIMAL_DIG__
  define: __FINITE_MATH_ONLY__=0
  define: __FLT_DECIMAL_DIG__=9
  define: __FLT_DENORM_MIN__=1.40129846e-45F
  define: __FLT_DIG__=6
  define: __FLT_EPSILON__=1.19209290e-7F
  define: __FLT_EVAL_METHOD__=0
  define: __FLT_HAS_DENORM__=1
  define: __FLT_HAS_INFINITY__=1
  define: __FLT_HAS_QUIET_NAN__=1
  define: __FLT_MANT_DIG__=24
  define: __FLT_MAX_10_EXP__=38
  define: __FLT_MAX_EXP__=128
  define: __FLT_MAX__=3.40282347e+38F
  define: __FLT_MIN_10_EXP__=(-37)
  define: __FLT_MIN_EXP__=(-125)
  define: __FLT_MIN__=1.17549435e-38F
  define: __FLT_RADIX__=2
  define: __FXSR__=1
  define: __GCC_ASM_FLAG_OUTPUTS__=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_1=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_2=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_4=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_8=1
  define: __INT16_C_SUFFIX__=
  define: __INT16_FMTd__="hd"
  define: __INT16_FMTi__="hi"
  define: __INT16_MAX__=32767
  define: __INT16_TYPE__=short
  define: __INT32_C_SUFFIX__=
  define: __INT32_FMTd__="d"
  define: __INT32_FMTi__="i"
  define: __INT32_MAX__=2147483647
  define: __INT32_TYPE__=int
  define: __INT64_C_SUFFIX__=LL
  define: __INT64_FMTd__="lld"
  define: __INT64_FMTi__="lli"
  define: __INT64_MAX__=9223372036854775807LL
  define: __INT64_TYPE__=long long int
  define: __INT8_C_SUFFIX__=
  define: __INT8_FMTd__="hhd"
  define: __INT8_FMTi__="hhi"
  define: __INT8_MAX__=127
  define: __INT8_TYPE__=signed char
  define: __INTMAX_C_SUFFIX__=LL
  define: __INTMAX_FMTd__="lld"
  define: __INTMAX_FMTi__="lli"
  define: __INTMAX_MAX__=9223372036854775807LL
  define: __INTMAX_TYPE__=long long int
  define: __INTMAX_WIDTH__=64
  define: __INTPTR_FMTd__="lld"
  define: __INTPTR_FMTi__="lli"
  define: __INTPTR_MAX__=9223372036854775807LL
  define: __INTPTR_TYPE__=long long int
  define: __INTPTR_WIDTH__=64
  define: __INT_FAST16_FMTd__="hd"
  define: __INT_FAST16_FMTi__="hi"
  define: __INT_FAST16_MAX__=32767
  define: __INT_FAST16_TYPE__=short
  define: __INT_FAST32_FMTd__="d"
  define: __INT_FAST32_FMTi__="i"
  define: __INT_FAST32_MAX__=2147483647
  define: __INT_FAST32_TYPE__=int
  define: __INT_FAST64_FMTd__="lld"
  define: __INT_FAST64_FMTi__="lli"
  define: __INT_FAST64_MAX__=9223372036854775807LL
  define: __INT_FAST64_TYPE__=long long int
  define: __INT_FAST8_FMTd__="hhd"
  define: __INT_FAST8_FMTi__="hhi"
  define: __INT_FAST8_MAX__=127
  define: __INT_FAST8_TYPE__=signed char
  define: __INT_LEAST16_FMTd__="hd"
  define: __INT_LEAST16_FMTi__="hi"
  define: __INT_LEAST16_MAX__=32767
  define: __INT_LEAST16_TYPE__=short
  define: __INT_LEAST32_FMTd__="d"
  define: __INT_LEAST32_FMTi__="i"
  define: __INT_LEAST32_MAX__=2147483647
  define: __INT_LEAST32_TYPE__=int
  define: __INT_LEAST64_FMTd__="lld"
  define: __INT_LEAST64_FMTi__="lli"
  define: __INT_LEAST64_MAX__=9223372036854775807LL
  define: __INT_LEAST64_TYPE__=long long int
  define: __INT_LEAST8_FMTd__="hhd"
  define: __INT_LEAST8_FMTi__="hhi"
  define: __INT_LEAST8_MAX__=127
  define: __INT_LEAST8_TYPE__=signed char
  define: __INT_MAX__=2147483647
  define: __LDBL_DECIMAL_DIG__=17
  define: __LDBL_DENORM_MIN__=4.9406564584124654e-324L
  define: __LDBL_DIG__=15
  define: __LDBL_EPSILON__=2.2204460492503131e-16L
  define: __LDBL_HAS_DENORM__=1
  define: __LDBL_HAS_INFINITY__=1
  define: __LDBL_HAS_QUIET_NAN__=1
  define: __LDBL_MANT_DIG__=53
  define: __LDBL_MAX_10_EXP__=308
  define: __LDBL_MAX_EXP__=1024
  define: __LDBL_MAX__=1.7976931348623157e+308L
  define: __LDBL_MIN_10_EXP__=(-307)
  define: __LDBL_MIN_EXP__=(-1021)
  define: __LDBL_MIN__=2.2250738585072014e-308L
  define: __LITTLE_ENDIAN__=1
  define: __LONG_LONG_MAX__=9223372036854775807LL
  define: __LONG_MAX__=2147483647L
  define: __MMX__=1
  define: __NO_INLINE__=1
  define: __NO_MATH_INLINES=1
  define: __OBJC_BOOL_IS_BOOL=0
  define: __OPENCL_MEMORY_SCOPE_ALL_SVM_DEVICES=3
  define: __OPENCL_MEMORY_SCOPE_DEVICE=2
  define: __OPENCL_MEMORY_SCOPE_SUB_GROUP=4
  define: __OPENCL_MEMORY_SCOPE_WORK_GROUP=1
  define: __OPENCL_MEMORY_SCOPE_WORK_ITEM=0
  define: __ORDER_BIG_ENDIAN__=4321
  define: __ORDER_LITTLE_ENDIAN__=1234
  define: __ORDER_PDP_ENDIAN__=3412
  define: __PIC__=2
  define: __POINTER_WIDTH__=64
  define: __PRAGMA_REDEFINE_EXTNAME=1
  define: __PTRDIFF_FMTd__="lld"
  define: __PTRDIFF_FMTi__="lli"
  define: __PTRDIFF_MAX__=9223372036854775807LL
  define: __PTRDIFF_TYPE__=long long int
  define: __PTRDIFF_WIDTH__=64
  define: __REGISTER_PREFIX__=
  define: __SCHAR_MAX__=127
  define: __SEG_FS=1
  define: __SEG_GS=1
  define: __SHRT_MAX__=32767
  define: __SIG_ATOMIC_MAX__=2147483647
  define: __SIG_ATOMIC_WIDTH__=32
  define: __SIZEOF_DOUBLE__=8
  define: __SIZEOF_FLOAT__=4
  define: __SIZEOF_INT128__=16
  define: __SIZEOF_INT__=4
  define: __SIZEOF_LONG_DOUBLE__=8
  define: __SIZEOF_LONG_LONG__=8
  define: __SIZEOF_LONG__=4
  define: __SIZEOF_POINTER__=8
  define: __SIZEOF_PTRDIFF_T__=8
  define: __SIZEOF_SHORT__=2
  define: __SIZEOF_SIZE_T__=8
  define: __SIZEOF_WCHAR_T__=2
  define: __SIZEOF_WINT_T__=2
  define: __SIZE_FMTX__="llX"
  define: __SIZE_FMTo__="llo"
  define: __SIZE_FMTu__="llu"
  define: __SIZE_FMTx__="llx"
  define: __SIZE_MAX__=18446744073709551615ULL
  define: __SIZE_TYPE__=long long unsigned int
  define: __SIZE_WIDTH__=64
  define: __SSE2_MATH__=1
  define: __SSE2__=1
  define: __SSE_MATH__=1
  define: __SSE__=1
  define: __STDC_HOSTED__=1
  define: __STDC_UTF_16__=1
  define: __STDC_UTF_32__=1
  define: __STDC_VERSION__=201710L
  define: __UINT16_C_SUFFIX__=
  define: __UINT16_FMTX__="hX"
  define: __UINT16_FMTo__="ho"
  define: __UINT16_FMTu__="hu"
  define: __UINT16_FMTx__="hx"
  define: __UINT16_MAX__=65535
  define: __UINT16_TYPE__=unsigned short
  define: __UINT32_C_SUFFIX__=U
  define: __UINT32_FMTX__="X"
  define: __UINT32_FMTo__="o"
  define: __UINT32_FMTu__="u"
  define: __UINT32_FMTx__="x"
  define: __UINT32_MAX__=4294967295U
  define: __UINT32_TYPE__=unsigned int
  define: __UINT64_C_SUFFIX__=ULL
  define: __UINT64_FMTX__="llX"
  define: __UINT64_FMTo__="llo"
  define: __UINT64_FMTu__="llu"
  define: __UINT64_FMTx__="llx"
  define: __UINT64_MAX__=18446744073709551615ULL
  define: __UINT64_TYPE__=long long unsigned int
  define: __UINT8_C_SUFFIX__=
  define: __UINT8_FMTX__="hhX"
  define: __UINT8_FMTo__="hho"
  define: __UINT8_FMTu__="hhu"
  define: __UINT8_FMTx__="hhx"
  define: __UINT8_MAX__=255
  define: __UINT8_TYPE__=unsigned char
  define: __UINTMAX_C_SUFFIX__=ULL
  define: __UINTMAX_FMTX__="llX"
  define: __UINTMAX_FMTo__="llo"
  define: __UINTMAX_FMTu__="llu"
  define: __UINTMAX_FMTx__="llx"
  define: __UINTMAX_MAX__=18446744073709551615ULL
  define: __UINTMAX_TYPE__=long long unsigned int
  define: __UINTMAX_WIDTH__=64
  define: __UINTPTR_FMTX__="llX"
  define: __UINTPTR_FMTo__="llo"
  define: __UINTPTR_FMTu__="llu"
  define: __UINTPTR_FMTx__="llx"
  define: __UINTPTR_MAX__=18446744073709551615ULL
  define: __UINTPTR_TYPE__=long long unsigned int
  define: __UINTPTR_WIDTH__=64
  define: __UINT_FAST16_FMTX__="hX"
  define: __UINT_FAST16_FMTo__="ho"
  define: __UINT_FAST16_FMTu__="hu"
  define: __UINT_FAST16_FMTx__="hx"
  define: __UINT_FAST16_MAX__=65535
  define: __UINT_FAST16_TYPE__=unsigned short
  define: __UINT_FAST32_FMTX__="X"
  define: __UINT_FAST32_FMTo__="o"
  define: __UINT_FAST32_FMTu__="u"
  define: __UINT_FAST32_FMTx__="x"
  define: __UINT_FAST32_MAX__=4294967295U
  define: __UINT_FAST32_TYPE__=unsigned int
  define: __UINT_FAST64_FMTX__="llX"
  define: __UINT_FAST64_FMTo__="llo"
  define: __UINT_FAST64_FMTu__="llu"
  define: __UINT_FAST64_FMTx__="llx"
  define: __UINT_FAST64_MAX__=18446744073709551615ULL
  define: __UINT_FAST64_TYPE__=long long unsigned int
  define: __UINT_FAST8_FMTX__="hhX"
  define: __UINT_FAST8_FMTo__="hho"
  define: __UINT_FAST8_FMTu__="hhu"
  define: __UINT_FAST8_FMTx__="hhx"
  define: __UINT_FAST8_MAX__=255
  define: __UINT_FAST8_TYPE__=unsigned char
  define: __UINT_LEAST16_FMTX__="hX"
  define: __UINT_LEAST16_FMTo__="ho"
  define: __UINT_LEAST16_FMTu__="hu"
  define: __UINT_LEAST16_FMTx__="hx"
  define: __UINT_LEAST16_MAX__=65535
  define: __UINT_LEAST16_TYPE__=unsigned short
  define: __UINT_LEAST32_FMTX__="X"
  define: __UINT_LEAST32_FMTo__="o"
  define: __UINT_LEAST32_FMTu__="u"
  define: __UINT_LEAST32_FMTx__="x"
  define: __UINT_LEAST32_MAX__=4294967295U
  define: __UINT_LEAST32_TYPE__=unsigned int
  define: __UINT_LEAST64_FMTX__="llX"
  define: __UINT_LEAST64_FMTo__="llo"
  define: __UINT_LEAST64_FMTu__="llu"
  define: __UINT_LEAST64_FMTx__="llx"
  define: __UINT_LEAST64_MAX__=18446744073709551615ULL
  define: __UINT_LEAST64_TYPE__=long long unsigned int
  define: __UINT_LEAST8_FMTX__="hhX"
  define: __UINT_LEAST8_FMTo__="hho"
  define: __UINT_LEAST8_FMTu__="hhu"
  define: __UINT_LEAST8_FMTx__="hhx"
  define: __UINT_LEAST8_MAX__=255
  define: __UINT_LEAST8_TYPE__=unsigned char
  define: __USER_LABEL_PREFIX__=
  define: __VERSION__="Clang 12.0.0"
  define: __WCHAR_MAX__=65535
  define: __WCHAR_TYPE__=unsigned short
  define: __WCHAR_UNSIGNED__=1
  define: __WCHAR_WIDTH__=16
  define: __WINT_MAX__=65535
  define: __WINT_TYPE__=unsigned short
  define: __WINT_UNSIGNED__=1
  define: __WINT_WIDTH__=16
  define: __amd64=1
  define: __amd64__=1
  define: __clang__=1
  define: __clang_major__=12
  define: __clang_minor__=0
  define: __clang_patchlevel__=0
  define: __clang_version__="12.0.0 "
  define: __code_model_small__=1
  define: __k8=1
  define: __k8__=1
  define: __llvm__=1
  define: __pic__=2
  define: __seg_fs=__attribute__((address_space(257)))
  define: __seg_gs=__attribute__((address_space(256)))
  define: __tune_k8__=1
  define: __x86_64=1
  define: __x86_64__=1
  define: _DEBUG
  define: _DEBUGBLD
  define: _CORTEX_A
  define: _SYSBLD
  other: --clang
  other: --clang_version=120000
  other: --ms_compatibility
  stdver: c17
  intelliSenseMode: linux-clang-x64
Checking for syntax errors: ---
Queueing IntelliSense update for files in translation unit of: ---
cpptools/finishUpdateSquiggles
Error squiggle count: 421
Tag Parser will be used for IntelliSense operations in: ---
Update IntelliSense time (sec): 1.346
  Discovering files: 69504 file(s) processed
  0 file(s) removed from database
Done discovering files.
Populating include completion cache.
Parsing remaining files...
  tag parsing file: ---

Then there are a couple hundred of these lines
Tag parsing encountered an error, but it may not matter. Let us know if symbols in the file can't be found: ---
....
Tag parsing encountered an error, but it may not matter. Let us know if symbols in the file can't be found: ---
  Parsing: 30434 files(s) processed
Done parsing remaining files.
textDocument/hover: ---
textDocument/documentHighlight: ---
cpptools/textEditorSelectionChange
cpptools/getCodeActions: ---

**Comment by Colengms** [comment]
We think this is a duplicate of https://github.com/microsoft/vscode-cpptools/issues/8851 .  There is some additional guidance in that issue.  We're in the process of adding some logging to getting better information on why this is failing.  If there is anything special about your system that might explain why this behavior is occurring (i.e. antivirus preventing access to those files, etc.), could you comment on that issue?

Resolving this as a duplicate.  If that gets addressed and this still repro's for you, we can reactivate.

**Comment by sean-mcmanus** [comment]
We've added tag parser error logging with 1.9.3 (Pre-Release) with C_Cpp.loggingLevel set to "Warning" or greater. You should see some text like
```
  tag parsing file: <path>
    tag parsing error (this can be ignored unless symbols can't be found):
    line 8 (col. 8): error: incorrectly formed universal character name
```
in the C/C++ logging pane. If you try that and let us know what error message is being shown that could help us fix the issue.

**Comment by nolancasimir** [comment]
> We've added tag parser error logging with 1.9.3 (Pre-Release) with C_Cpp.loggingLevel set to "Warning" or greater. You should see some text like
> 
> ```
>   tag parsing file: <path>
>     tag parsing error (this can be ignored unless symbols can't be found):
>     line 8 (col. 8): error: incorrectly formed universal character name
> ```
> 
> in the C/C++ logging pane. If you try that and let us know what error message is being shown that could help us fix the issue.

Thanks Sean, 
I get a ton of these errors

tag parsing file: <path>
    tag parsing error (this can be ignored unless symbols can't be found):
    Catastrophic error: internal error

**Comment by sean-mcmanus** [comment]
`Catastrophic error: internal error` -- yikes, I was worried it might show something like that :(

**Comment by nolancasimir** [comment]
I am not seeing it on an empty file currently.

**Comment by sean-mcmanus** [comment]
Can you try the vsix at https://github.com/microsoft/vscode-cpptools/releases/tag/v1.9.4-debug (i.e. https://github.com/microsoft/vscode-cpptools/releases/download/v1.9.4-debug/cpptools-win64.vsix ) and report the debug assertion message that occurs from the tag parsing? That info should help us fix this issue. It may look something like:

![image](https://user-images.githubusercontent.com/19859882/158285364-7aba2223-a785-4ae8-bcbc-4956caa7a1a0.png)

**Comment by sean-mcmanus** [comment]
I got the info I need from another user -- investigating it...

**Comment by sean-mcmanus** [comment]
FYI, the cpptools-win64.vsix at https://github.com/microsoft/vscode-cpptools/releases/tag/1.9.4-prerelease1 should have the issue fixed -- we're working on releasing 1.9.4 soon, but it should be very similar to that build.

**Comment by nolancasimir** [comment]
Thanks! I will give this a go!

**Comment by sean-mcmanus** [comment]
We've just released 1.9.4 (pre-release), but it's nearly identical to 1.9.4-prerelease1, which should be nearly identical to 1.9.5 (non-pre-release) planned for Monday.
